packages/docs/docs/zh/plugin/official/plugin-register-components.md

```diff
@@ -35,14 +35,15 @@ module.exports = {
 module.exports = {
   plugins: [
     [
-      'register-components',
+      '@vuepress/register-components',
       {
         componentsDir: somepath
       }
     ]
   ]
 }
 ```
+`somepath`是从`docs`目录开始,比如：`docs/.vuepress/components/custom-path/`
```

Comment:
It depends. The directory name is not required to be `docs`.

Reply:
So `somepath` might've confused you. That's why you want to add an additional line, right?

Thanks a lot for your help to improve docs. Could you also update the EN docs for this line?
Besides, I'd suggest placing them right after `somepath` as comment. 

Or you can leave it to me and just remove this line to let this PR focus on fixing example error.

Reply:
ok,somepath is deleted